code/game/objects/items/plushes.dm

```diff
@@ -676,3 +676,23 @@
 	item_state = "moffplush"
 	attack_verb = list("fluttered", "flapped")
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+	var/souls = 0
+
+/obj/item/toy/plush/moth/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] begins staring deeply into the eyes of [src]!  It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	if(do_after(user, 50, target = user) && !user.hellbound)
+		souls++
+		switch(souls)
+			if(1 to 2)
+				desc = "A plushie depicting an unsettling mothperson. After eating [souls] [souls == 1 ? "soul" : "souls"] it's not looking so huggable now..."
+			if(3 to INFINITY)
+				desc = "A plushie depicting a creepy mothperson. It's eaten [souls] souls! I don't think I want to hug it any more!"
+				divine = TRUE
+				resistance_flags = INDESTRUCTIBLE | FIRE_PROOF | ACID_PROOF | LAVA_PROOF
+		user.visible_message("<span class='suicide'>[src] rips the soul out of [user] and consumes it!</span>")
+		playsound(src, 'sound/hallucinations/wail.ogg', 50, TRUE, -1)
+		user.unequip_everything()
+		user.dust()
```

Comment:
dust has arguments

Reply:
https://github.com/tgstation/tgstation/blob/7517c03dba1f75e45287b26efe3efe8e07ce6c84/code/modules/mob/death.dm#L9

Reply:
the second one 👀